cxplat/inc/cxplat_memory.h

```diff
@@ -113,4 +113,46 @@ cxplat_duplicate_string(_In_z_ const char* source);
 void
 cxplat_free_string(_Frees_ptr_opt_ _In_z_ const char* string);
 
+/**
+ * @brief Initialize a lookaside list.
+ *
+ * @param[out] lookaside The initialized lookaside list.
+ * @param[in] size Allocation size of each entry in the lookaside list.
+ * @param[in] tag Tag to use for the lookaside list.
+ * @param[in] pool_flags Flags to use for the lookaside list.
+ * @param[in] allocate Optional allocation callback.
+ * @param[in] free Optional free callback.
+ * @retval CXPLAT_STATUS_SUCCESS The operation was successful.
+ * @retval CXPLAT_STATUS_NO_MEMORY Unable to allocate resources for this lookaside list.
+ */
+_Must_inspect_result_ cxplat_status_t
+cxplat_initialize_lookaside_list(
+    _Out_ cxplat_lookaside_list_t* lookaside, _In_ size_t size, _In_ uint32_t tag, _In_ cxplat_pool_flags_t pool_flags);
+
+/**
+ * @brief Free a lookaside list.
+ *
+ * @param[in] lookaside Lookaside list to free.
+ */
+void
+cxplat_uninitialize_lookaside_list(_Inout_ cxplat_lookaside_list_t* lookaside);
+
+/**
+ * @brief Allocate an entry from a lookaside list.
+ *
+ * @param[in,out] lookaside Lookaside list to allocate from.
+ * @return Allocation from the lookaside list, or NULL if the allocation failed.
+ */
+_Must_inspect_result_ void*
+cxplat_lookaside_list_alloc(_Inout_ cxplat_lookaside_list_t* lookaside);
```

Comment:
```suggestion
cxplat_allocate_lookaside_list_entry(_Inout_ cxplat_lookaside_list_t* lookaside);
```
Verb first, like line 129 does